src/services/user/services/AdminUsers.js

```diff
@@ -173,6 +176,7 @@ class AdminUsers {
 		const users = await this.prepareRoleback(email, _id, () =>
 			this.app.service('usersModel').patch(null, _data, params)
 		);
+		await this.markUserAsDeleted(email, _id);
```

Comment:
better use delete route for deleting the user

Reply:
done ;)

Reply:
done ;)